lib/ex_css_modules/view.ex

```diff
@@ -19,7 +20,14 @@ defmodule ExCSSModules.View do
 
   defmacro __using__(opts \\ []) do
     {filename, [file: relative_to]} = Code.eval_quoted(opts[:stylesheet], file: __CALLER__.file)
-    {embed, _} = Code.eval_quoted(opts[:embed_stylesheet])
+
+    embed =
+      Keyword.get(
+        opts,
+        :embed_stylesheet,
+        Application.get_env(:ex_css_modules, :embed_by_default, false)
+      )
```

Comment:
Think we'd want to have this **inside** the `quote` block, so it can retrieve the Application.env value at runtime (to support runtime configs in pre-compiled Mix Release / Distillery apps)

Reply:
Isn't the point of the embed option to compile the stylesheet into the module?

I'm having trouble seeing how this could work at runtime short of having some sort of external cache like an agent or something :thinking:

Reply:
Sorry, you're right, doesn't make much sense indeed. Been working too much on compile time vs runtime config issues lately, my head was somewhere else :P

I'll test this PR in our own app somewhere this week and will get back to you.